obal/data/modules/check_koji_build.py

```diff
@@ -0,0 +1,39 @@
+"""
+Check if build exists in Koji
+"""
+
+import os
+from subprocess import STDOUT, check_output
+
+from ansible.module_utils.basic import AnsibleModule
+from ansible.module_utils.obal import get_specfile_nevr
+
+def main():
+    """
+    Check if build exists in Koji
+    """
+    module = AnsibleModule(
+        argument_spec=dict(
+            tag=dict(type='str', required=True),
+            nevr=dict(type='str', required=True),
+            package=dict(type='str', required=True)
+        )
+    )
+
+    nevr = module.params['nevr']
+    tag = module.params['tag']
+    package = module.params['package']
+
+    command = ['koji', 'latest-build', '--quiet', tag, package]
+
+    try:
+        build = check_output(command, stderr=STDOUT, universal_newlines=True)
+        build = build.split(' ')[0]
+
+        module.exit_json(changed=False, tagged_version=build, exists=(nevr == build))
```

Comment:
the build contains `%dist`, while `nevr` does not (as we do `--undefine dist` when querying it), so this will never be true.

Reply:
or do you plan to pass `dist` from the tag definition in the manifest, and it's just not in the examples below?

Reply:
Ahh yes, sorry about that. I forgot to update the fixtures with this push to represent the proposed structure. Next push will include that update.